### PR TITLE
Include tests for importing metadata - /dsmetadata API 

### PR DESCRIPTION
**Pull request by shreyabiradar07**
This PR implements test cases for `/dsmetadata` API

### CODE REVIEW

File: tests/scripts/functional_tests.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ SCRIPTS_DIR="${CURRENT_DIR}"
 . ${SCRIPTS_DIR}/da/kruize_layer_id_tests.sh
 . ${SCRIPTS_DIR}/em/em_standalone_tests.sh
 . ${SCRIPTS_DIR}/remote_monitoring_tests/remote_monitoring_tests.sh
+. ${SCRIPTS_DIR}/local_monitoring_tests/local_monitoring_tests.sh
```
Comment (chandrams) [inline tests/scripts/functional_tests.sh:35]: Can you please check if we need updates to test_autotune.sh and update it as well.

Comment (chandrams) [reply tests/scripts/functional_tests.sh:35]: Update relevant test docs as well

File: tests/scripts/local_monitoring_tests/helpers/fixtures.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import pytest
```
Comment (chandrams) [inline tests/scripts/local_monitoring_tests/helpers/fixtures.py:1]: If helpers are common between local monitoring and remote monitoring, consider having a single copy

File: tests/scripts/local_monitoring_tests/rest_apis/test_import_metadata.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    response = import_metadata(input_json_file)
+
+    data = response.json()
+    print("response data: ", str(data))
+    print(data['message'])
+
+    assert response.status_code == SUCCESS_STATUS_CODE
+    assert data['status'] == SUCCESS_STATUS
+    assert data['message'] == IMPORT_METADATA_SUCCESS_MSG
+
```
Comment (chandrams) [inline tests/scripts/local_monitoring_tests/rest_apis/test_import_metadata.py:36]: Please add validation for the json output received

File: tests/scripts/local_monitoring_tests/helpers/fixtures.py (hunk 2 above)
Comment (shreyabiradar07) [reply tests/scripts/local_monitoring_tests/helpers/fixtures.py:1]: helpers has common files which can be shared, except for - `generate_rm_jsons.py`, wherein both remote and local `target_cluster` need to be supported 
Will it be okay to add a new parameter in [create_exp_jsons()](https://github.com/shreyabiradar07/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/helpers/generate_rm_jsons.py#L34) to specify the target_cluster?
This way a single copy of helpers can be used

Comment (chandrams) [reply tests/scripts/local_monitoring_tests/helpers/fixtures.py:1]: Yes, but make sure the remote monitoring tests aren't affected

Comment (shreyabiradar07) [reply tests/scripts/local_monitoring_tests/helpers/fixtures.py:1]: [create_exp_jsons()](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/helpers/generate_rm_jsons.py#L34) is used by [kruize_pod_restart.py](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/fault_tolerant_tests/kruize_pod_restart_test.py#L91) (fault tolerant tests) and [test_list_recommendations.py](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py#L459)
adding a new parameter to [create_exp_jsons()](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py#L459) might affect remote monitoring tests

Comment (chandrams) [reply tests/scripts/local_monitoring_tests/helpers/fixtures.py:1]: You can add a default value to the parameter as remote and override it for local tests.

**Comment by shreyabiradar07** [comment]
Raised  [#1187](https://github.com/kruize/autotune/pull/1187) including above changes of local monitoring test suite. Closing this in favour of PR 1187

**Pull request closed without merging**